timemachine/integrator.py

```diff
@@ -154,14 +164,17 @@ def __init__(self, force_fxn, masses, dt):
         self.dt = dt
         self.masses = masses[:, np.newaxis]  # TODO: cleaner way to handle (n_atoms,) vs. (n_atoms, 3) mismatch?
         self.force_fxn = force_fxn
+        self.cb = self.dt / self.masses
 
     def step(self, x, v):
         """WARNING: makes 2 calls to force_fxn per timestep -- prefer `.multiple_steps` in most cases"""
-        v_mid = v + (0.5 * self.dt / self.masses) * self.force_fxn(x)
-        x = x + self.dt * v_mid
-        v = v_mid + (0.5 * self.dt / self.masses) * self.force_fxn(x)
+        fixed_x = FLOAT_TO_FIXED(x)
+        fixed_v = FLOAT_TO_FIXED(v)
+        v_mid = fixed_v + FLOAT_TO_FIXED((0.5 * self.cb) * self.force_fxn(x))
+        fixed_x = fixed_x + FLOAT_TO_FIXED(self.dt * FIXED_TO_FLOAT(v_mid))
+        fixed_v = v_mid + FLOAT_TO_FIXED((0.5 * self.cb) * self.force_fxn(FIXED_TO_FLOAT(fixed_x)))
```

Comment:
would this be easier to read:
```
v_mid = FLOAT_TO_FIXED(v) + FLOAT_TO_FIXED((0.5 * self.cb) * self.force_fxn(x))
fixed_x = FLOAT_TO_FIXED(x) + FLOAT_TO_FIXED(self.dt * FIXED_TO_FLOAT(v_mid))
fixed_v = v_mid + FLOAT_TO_FIXED((0.5 * self.cb) * self.force_fxn(FIXED_TO_FLOAT(fixed_x)))
```
I was trying to figure out where `fixed_x` was being used since it was declared twice